docs/advanced/vm-migration-network.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 12
+sidebar_label: VM Migration Network
+title: "VM Migration Network"
+---
+
+If the user wishes to isolate VM migration traffic from the Kubernetes cluster network (i.e. the management network) or other cluster-wide workloads. Users can allocate a dedicated vm migration network to get better network bandwidth and performance.
+
+:::note
+
+- Avoid configuring KubeVirt configuration directly, as this can result in unexpected or unwanted system behavior.
+
+:::
+
+## Prerequisites
+
+There are some prerequisites before configuring the Harvester VM Migration Network setting.
+
+- Well-configured Cluster Network and VLAN Config.
+    - Users have to ensure the Cluster Network is configured and VLAN Config will cover all nodes and ensure the network connectivity is working and expected in all nodes.
+- No VM Migration in progress before configuring the VM Migration Network setting.
+
+:::caution
+
+If the Harvester cluster was upgraded from v1.0.3, please check if Whereabouts CNI is installed properly before you move on to the next step. We will always recommend following this guide to check. [Issue 3168](https://github.com/harvester/harvester/issues/3168) describes that the Harvester cluster will not always install Whereabouts CNI properly.
+
+- Verify the `ippools.whereabouts.cni.cncf.io` CRD exists with the following command.
+    - `kubectl get crd ippools.whereabouts.cni.cncf.io`
+- If the Harvester cluster doesn't have `ippools.whereabouts.cni.cncf.io`, please add [these two CRDs](https://github.com/harvester/harvester/tree/v1.1.0/deploy/charts/harvester/dependency_charts/whereabouts/crds) before configuring `storage-network` setting.
```

Comment:
Why suggest using the CRDs in 1.1.0 instead of the corresponding Harvester release? Might just copy from the storage network?

Reply:
Remove this since we already have ippools in Harvester.